src/app/[locale]/components/Footer.tsx

```diff
@@ -0,0 +1,61 @@
+'use client'
+
+import { Link } from '@utrecht/component-library-react'
+import { IconChevronRight } from '@tabler/icons-react'
+import { useEffect, useState } from 'react'
+import { useConfig } from '@/hooks/useConfig'
+import { useTranslations } from 'next-intl'
+
+type footerLink = {
+  href: string
+  label: string
+}
+
+const Footer = () => {
+  const [links, setLinks] = useState<footerLink[]>([])
+  const { config, loading } = useConfig()
+  const t = useTranslations('footer')
+
+  useEffect(() => {
+    if (!loading && config) {
+      const newLinks: footerLink[] = []
+
+      if (config.base.links['about']) {
```

Comment:
Is het een idee om dit dynamischer te maken? Zodat het niet enkel bij een mogelijke "Over", "Privacy" en "Toegankelijkheid" optie blijft?

Ik denk persoonlijk meer aan de optie dat de beheerder gewoon binnen het config.base.links object zelf objecten kan toevoegen met een label en link en we daarover heen loopen.